ports/kf5codecs/vcpkg.json

```
@@ -1,11 +1,12 @@
 {
   "name": "kf5codecs",
   "version": "5.84.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "String encoding library",
   "homepage": "https://api.kde.org/frameworks/kcodecs/html/index.html",
   "dependencies": [
     "ecm",
+    "gperf",
```

```suggestion
    {
      "name": "gperf",
      "host": true
    }
```

It looks like this intends to run gperf so it should be a host dependency?

---

Good call, thanks.

---

Well, now it fails with:
```
CMake Error at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Gperf (missing: Gperf_EXECUTABLE)
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  D:/installed/x64-windows-static/share/ECM/find-modules/FindGperf.cmake:60 (find_package_handle_standard_args)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:786 (_find_package)
  CMakeLists.txt:23 (find_package)
```

---

Maybe I should instead:
1) Bump win32 `gperf` version and add a `BREW_PACKAGE_NAME`: https://github.com/microsoft/vcpkg/blob/8275003fd8c5119cb9438fea1425ebe774f2e527/scripts/cmake/vcpkg_find_acquire_program.cmake#L333
2) Add a warning to kf5codec regarding minimum `gperf` version required

---

gperf installs these files:
~~~
gperf:/share/gperf/copyright
gperf:/share/gperf/vcpkg_abi_info.txt
gperf:/tools/gperf.exe
~~~
There is no `vcpkg-cmake-config.cmake`, so there is nothing magically injecting the host binary path to the `FindGperf.cmake` search process. (Do you remember `gettext[tools]`? Or #17607?)
But you can probaby preset the desired `Gperf_EXECUTABLE` cache variable in `portfile.cmake`.

---

@dg0yt I just checked on macOS here and it does find it just fine in `tools/gperf` without any extra steps, so this is something with Windows:

```
-- Found Gperf: /Users/cromo/Documents/Sourcecode/vcpkg/installed/arm64-osx/tools/gperf
```

---

On your mac, is host triplet = target triplet?
On a standard modern windows PC, the triplets differ.